File: src/main/java/no/nav/veilarbaktivitet/avtalt_med_nav/ForhaandsorienteringDTO.java
package no.nav.veilarbaktivitet.avtalt_med_nav;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

import java.util.Date;


@Data
@Builder(toBuilder = true)
@AllArgsConstructor
public class ForhaandsorienteringDTO {
    private String id;
    private Type type;
    private String tekst;
    private Date lestDato;
}

File: src/main/kotlin/no/nav/veilarbaktivitet/avtalt_med_nav/ForhaandsorienteringDTO.kt
package no.nav.veilarbaktivitet.avtalt_med_nav

import java.util.*

data class ForhaandsorienteringDTO(
    val id: String,
    val type: Type,
    val tekst: String,
    val lestDato: Date?,
)

data class ForhaandsorienteringInboundDTO(
    val type: Type,
    val tekst: String,
)
